examples/test_sph_harm_poles.cc

```diff
@@ -1,5 +1,5 @@
 /**
- * file   test_sph_harm_poles.cc
+ * @file   examples/test_sph_harm_poles.cc
```

Comment:
I thought we'd agreed that this file can be deleted. When did it get revived?

Reply:
It is deleted in #194